### ISSUE TITLE
As a TTS organization, I want standardization across all AWS accounts

### ISSUE THREAD

**Issue by JJediny** [18F/tts-tech-portfolio#70]
During the process of taking over ownership of all TTS AWS accounts a few inconsistencies were found, this issue is to track their standardization:

- [ ] create new dedicated/shared {{accountnumber}}@gsa.gov emails for remaining accounts w/o.
   - [ ] Search.gov root user is a POC
   - [ ] Data.gov sandbox root user is a POC
   - [ ] OPP Production (cloudfront/s3 only) root user is a POC
- [ ] establish a standard naming convention for account shortnames. ( e.g. GSA-FAS-TTS-OPP-{{system}}-{{env}}, GSA-FAS-TTS-PIF-{{system}}-{{env}}, GSA-FAS-TTS-18f-{{system}}-{{env}}, GSA-FAS-TTS-COE-{{system}}-{{env}})
- [ ] ensure all govcloud accounts are well documented for accessing. 
- [ ] ensure TTS Tech portfolio members authenticator apps are well associated with the account numbers. *until federated login is addressed.

**Comment by afeld** [comment]
Blocked by #76.

**Comment by afeld** [comment]
Let's split this into smaller independent tasks.

**Comment by JJediny** [comment]
before splitting need to work with cloud.gov/login.gov/OPP teams on conscience approach. Still need to 
- [ ] prepare agenda and doc to review scope
- [ ] schedule a meeting
- [ ] split issue out based on feedback during last portion of meeting

**Comment by afeld** [comment]
> create new dedicated/shared {{accountnumber}}@gsa.gov emails

Just realized we can use `devops+<accountnumber>@gsa.gov`, which is a lot easier than setting up individual groups. I suggest we go with that going forward.

**Comment by afeld** [comment]
> establish a standard naming convention for account shortnames

We can assume they are all under TTS, so can drop that prefix.

**Comment by afeld** [comment]
cc https://github.com/18F/aws-admin/issues/2

**Comment by JJediny** [comment]
Current documentation/report out needs review:

aws-admin scoping:
https://docs.google.com/document/d/1vwlrn-WsmqYsa36T98sBgD0W_JRyLyboa-dQydYw4Go/edit

AWS and login.gov Analysis:
https://docs.google.com/document/d/1nO0UIbi3DCkSVDUTLblKIEPmnJJt8Vt-A00w4QSYTmQ/edit

**Comment by its-a-lisa** [comment]
> <@U02QLQ88X> <@U0ELDTQ6L> thoughts on exploring the topic of a Trusted Cloud Credential Manager (TCCM) as described in the DoD Cloud Security Architecture?
<https://dl.dod.cyber.mil/wp-content/uploads/cloud/pdf/SCCA_FRD_v2-9.pdf>
<sub>[View in Slack](https://gsa-tts.slack.com/archives/C039MHHF8/p1570739518026200)</sub>
